core/storage/topic/gae_models.py

```diff
@@ -21,6 +21,150 @@
 from google.appengine.ext import ndb
 
 
+class TopicSnapshotMetadataModel(base_models.BaseSnapshotMetadataModel):
+    """Storage model for the metadata for a topic snapshot."""
+    pass
+
+
+class TopicSnapshotContentModel(base_models.BaseSnapshotContentModel):
+    """Storage model for the content of a topic snapshot."""
+    pass
+
+
+class TopicModel(base_models.VersionedModel):
+    """Model for storing Topics.
+
+    This class should only be imported by the topic domain file, the
+    topic services file, and the topic model test file.
+    """
+    SNAPSHOT_METADATA_CLASS = TopicSnapshotMetadataModel
+    SNAPSHOT_CONTENT_CLASS = TopicSnapshotContentModel
+    ALLOW_REVERT = True
```

Comment:
not sure this needs reverts, it's all just strings that can be easily changed to what they're meant to be if needed

Reply:
Done.